recipe/meta.yaml

```diff
@@ -11,45 +12,47 @@ source:
 
 build:
   noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+  number: 1
+  entry_points:
+    - datasets-cli=datasets.commands.datasets_cli:main
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python >=3.6
     - pip
+    - python >=3.6
   run:
     - aiohttp
-    - conllu
     - dataclasses
     - dill
-    - fsspec
-    - huggingface_hub <0.1.0
-    - importlib_metadata
-    - lxml
+    - fsspec >=2021.05.0
+    - huggingface_hub >=0.0.14,<0.1.0
+    - importlib-metadata
     - multiprocess
     - numpy >=1.17
-    - openpyxl
     - packaging
     - pandas
-    - py7zr
-    - pyarrow >=0.17.1
+    - pyarrow >=1.0.0,!=4.0.0
+    - requests >=2.19.0
+    - tqdm >=4.62.1
     - python >=3.6
     - python-xxhash
-    - requests >=2.19.0
-    - tqdm >=4.27,<4.50.0
 
 test:
   imports:
     - datasets
     - datasets.commands
     - datasets.utils
+  commands:
+    - pip check
+    - datasets-cli --help
+  requires:
+    - pip
 
 about:
   home: https://github.com/huggingface/datasets
   license: Apache-2.0
   license_family: Apache
-  # License file manually packaged. See: https://github.com/huggingface/datasets/pull/1007
   license_file: LICENSE
   summary: HuggingFace/Datasets is an open library of NLP datasets.
   doc_url: https://huggingface.co/docs/datasets/
```

Comment:
Lets add a `description` entry here, just an copy-and-paste (slightly edited) from upstream:
> Datasets is a lightweight library providing one-line dataloaders for many public datasets and one liners to download and pre-process any of the number of datasets major public datasets provided on the HuggingFace Datasets Hub. Datasets are ready to use in a dataloader for training/evaluating a ML model (Numpy/Pandas/PyTorch/TensorFlow/JAX). Datasets also provide an API for simple, fast, and reproducible data pre-processing for the above public datasets as well as your own local datasets in CSV/JSON/text.